docs-ref-conceptual/get-started-tutorial-3-use-variables.md

```diff
@@ -0,0 +1,305 @@
+description: Learn how to use variables to store JSON file properties and query output.
+manager: jasongroce
+author: dbradish-microsoft
+ms.author: dbradish
+ms.date: 11/15/2023
+ms.topic: conceptual
+ms.service: azure-cli
+ms.tool: azure-cli
+ms.custom: devx-track-azurecli
+keywords: azure, 
+---
+# Populate variables for use in scripts
+
+In this tutorial step you will learn how to get information from a local JSON file, new and existing Azure resources, and store the information in a variable. The variable can then be used in Azure CLI to perform build and destroy jobs at scale. Some common use-cases are:
+
+* Get information about an existing Azure resource, such as a resource ID.
+* Get output from an Azure CLI command, such as a password.
+* Get JSON objects for environment information, such as development, stage and production IDs.
+
+## Prerequisites
+
+* You have access to a resource group and storage account with `contributor` permissions.
+
+## Get command output using JMESPath query
+
+There are times when you want to get information about an existing Azure resource and return that information to your console screen, or store it in a variable for use within a script. In the Azure CLI, use the `--query` parameter to execute a [JMESPath query](https://jmespath.org/) to perform these tasks.
+
+> [!TIP]
+> The syntax for `--query` is case sensitive _and environment-specific_.  If you receive empty results, check your capitalization. Avoid quoting errors by applying the rules you learned in [Write Azure CLI commands for different environments](./get-started-tutorial-2-environment-syntax.md)
+
+Unless the `--output` parameter is specified, these examples rely on a default output configuration of `json` set in [Prepare your environment for the Azure CLI](./get-started-tutorial-1-prepare-environment.md)
+
+### Get JSON dictionary properties of an Azure resource
+
+Using the storage account created in [Learn Azure CLI syntax differences in Bash, PowerShell and Cmd](./get-started-tutorial-2-environment-syntax.md), get the `primaryEndpoints` of your new storage account.
+
+```azurecli-interactive
+az storage account show --resource-group <msdocs-tutorial-rg-00000000> \
+                        --name <msdocssa000000000> \
+                        --query primaryEndpoints
+```
+
+Console JSON dictionary output:
+
+```output
+{
+  "blob": "https://msdocssa00000000.blob.core.windows.net/",
+  "dfs": "https://msdocssa00000000.dfs.core.windows.net/",
+  "file": "https://msdocssa00000000.file.core.windows.net/",
+  "internetEndpoints": null,
+  "microsoftEndpoints": null,
+  "queue": "https://msdocssa00000000.queue.core.windows.net/",
+  "table": "https://msdocssa00000000.table.core.windows.net/",
+  "web": "https://msdocssa00000000.z13.web.core.windows.net/"
+}
+```
+
+### Get individual JSON objects
+
+```azurecli-interactive
+az storage account show --resource-group <msdocs-tutorial-rg-00000000> \
```

Comment:
When I run these lines i get: ERROR: unrecognized arguments: [id, primaryLocation, primaryEndpoints.blob, encryption.services.blob.lastEnabledTime]. I replaced the other fields with my own, but maybe there needs to be additional clarification for this code block? Not sure if it is something wrong on my end